plugins/livelog/livelog.go

```diff
@@ -100,11 +103,42 @@ func (tp *taskPlugin) Finished(success bool) error {
 		return err
 	}
 	defer file.Close()
+
+	tempDir, err := runtime.NewTemporaryStorage(os.TempDir())
```

Comment:
sorry, no...
In `NewPlugin` you get a copy of `runtime.Environment` from the options... Keep that, it contains an instance of temporary storage that you can use.